build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# runs source-deps and tests and then provided lein target with mranderson
+
+lein do clean, source-deps
+lein with-profile +plugin.mranderson/config test
+lein with-profile plugin.mranderson/config "$@"
```

Comment:
A while back I tried making leiningen aliases to work with `mranderson` but I failed miserably and created some bash scripts too.  

I still think it would be nice to get this all done in leiningen, though.